test/e2e/run_test.sh

```diff
@@ -226,6 +241,21 @@ while (( "$#" )); do
       SKIP_BUILD=true
       shift
     ;;
+    --use-cluster)
+      if [ -n "$2" ] && [[ "$2" != --* ]]; then
```

Comment:
There is no need to check whether the cluster name is correct, because the parameters don't just start with "--". If it is not correct, it will fail directly.